README.md

```diff
@@ -1,58 +1,68 @@
 # Project - EdEHR
-> Educational Electronic Health Record system: a [BCcampus](https://bccampus.ca) project
+> Educational Electronic Health Record system: a [BCcampus](https://bccampus.ca) project.
 
-## Description
+This project aims to create a student-focused Eletronic Health System where students and professors would be able to keep track of their 
+assessments. For more information on the project and it's guidelines, please refer to the [About](./about.md) section.
 
-EdEHR is an open source web application developed to teach medical students how to use an EHR before they enter the workforce. It can be accessed only through a learning management system (LMS) and requires a technical person to set it up. Once set up, users can access the system either as an instructor or a student. Credentials to try out either role are included below.
 
-## License
+## Documentation
+
+We encourage developers to read the documentation before contributing. Understanding the definitions and the architecture of the project will help you understand the codebase and make it easier for you to contribute to the project.
 
+For more details about this project, [view the documentation](https://edehr.github.io/edehr/)
+
+Alternatively, you can run a local documentation server by running the following command in your terminal:
+```
+npm run docs:dev
+```
+Then visit  [http://localhost:8080/edehr](http://localhost:8080/edehr) See documentation in the ```docs``` directory.
 
-The master-bccampus branch is released under a copyright licence (MIT). This includes all work funded by BC Campus from project start, September 2018 to project end, January 31, 2020.
```

Comment:
Keep these two lines even if the text has been copied somewhere else.  This root readme needs to present the essentials of the project and the licensing is one of these essentials.